internal/resources/google/compute_vpn_gateway.go

```diff
@@ -0,0 +1,42 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/usage"
+)
+
+type ComputeVpnGateway struct {
```

Comment:
```suggestion
type ComputeVPNGateway struct {
```
Sorry, I think I missed this case before, I wonder if we can add this to the linter somehow.